### PR TITLE
add generated notes from today's educators meeting

### PR DESCRIPTION
**Pull request by bkardell**

### CODE REVIEW

File: educators/notes-2019-04-11.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* In theory we know what we want to cover
+* Assign a content-creator and champion
+* They meet for 30-60 minutes
+* We provide a step-by-step process/framework
+* Start with process
+* Move on to Stage 3
+* Ultimately require to advance out of Stage 2
+
+
+The group is well aware that this meeting is not an official part of TC39, but is content to meet anyway and discuss these issues. All attendees were interested in continuing to meet monthly, for an hour, with a new Doodle each month to accommodate attendees' complex schedules.
```
Comment (littledan) [inline educators/notes-2019-04-11.md]: Is this line part of what was discussed this meeting, or was it from earlier?

**Comment by bkardell** [comment]
Ah no lol..that is from earlier..i thought maybe it was a disclaimer you wanted to keep in there

**Comment by littledan** [comment]
No problem; could you just remove it?

**Comment by bkardell** [comment]
Should be good now?  Sorry about that.

**Pull request merged**